lib/services/android-project-service.ts

```diff
@@ -209,16 +209,17 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		// will replace applicationId in app/App_Resources/Android/app.gradle if it has not been edited by the user
 		const userAppGradleFilePath = path.join(projectData.appResourcesDirectoryPath, this.$devicePlatformsConstants.Android, "app.gradle");
 
+		//TODO: kddimitrov: For compatability with old templates. Once all templates are updated should delete.
 		try {
-			shell.sed('-i', /__PACKAGE__/, projectData.projectId, userAppGradleFilePath);
+			shell.sed('-i', /__PACKAGE__/, projectData.projectIdentifiers.android, userAppGradleFilePath);
 		} catch (e) {
-			this.$logger.warn(`\n${e}.\nCheck if you're using an outdated template and update it.`);
+			// Ignore
```

Comment:
I think we should at least trace here, so we don't forget to remove this code later on

Reply:
Well I left a comment to remind us to remove this peace of code. If you consider tracing is better I can remove the comment. Do you prefer both the comment and trace?

Reply:
Personally, I prefer trace, because it see it anytime I'm debugging as opposed to looking at the code (not so often)